external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopVillainsIT.java

```diff
@@ -43,7 +43,7 @@
     private static final String POSTGRESQL_DATABASE = "villains-database";
     private static final int POSTGRESQL_PORT = 5432;
 
-    @Container(image = "registry.redhat.io/rhscl/postgresql-12-rhel7", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
+    @Container(image = "${postgresql.10.image}", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
```

Comment:
PostgreSQL 12 should be used

Reply:
Ok, then I will mark this scenario with `@EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")`